### DISCUSSION TITLE
Windows insider build failing

### DISCUSSION THREAD

**Discussion by jcasale** [VSCodium/vscodium#1380] [General]
I was looking for info on the error plaguing Windows insider builds and found [#1375](https://github.com/VSCodium/vscodium/issues/1375#issuecomment-1380989727).

Does anyone know what needs to be done to resolve that issue?

Thanks.

**Comment by daiyam** [comment]
Here are update: https://github.com/VSCodium/vscodium/pull/1370#issuecomment-1399300420

**Comment by daiyam** [comment]
It's fixed
